### PR TITLE
[Tabbed Application Mode article] Add `"tab_strip"` details 

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add `"tab_strip"` details.

### CODE REVIEW

File: src/site/content/en/blog/tabbed-application-mode/index.md [hunk 1]
```diff
@@ -108,10 +104,75 @@ To use tabbed application mode, developers need to opt their apps in by setting
...
 {
-  …
   "display": "standalone",
+  "display_override": ["tabbed"]
+}
+```
+
+Next, the property `"tab_strip"` can optionally be used to fine-tune the tab behavior. It has two
+allowed sub-properties, `"home_tab"` and `"new_tab_button"`. If the `"tab_strip"` property is not
+present, the particular properties' `"auto"` values are used. The browser can decide what values to
```
Comment (jpmedley) [inline src/site/content/en/blog/tabbed-application-mode/index.md:114]:
```suggestion
present, the particular properties' `"auto"` values are used. The browser determans what values to
```

File: src/site/content/en/blog/tabbed-application-mode/index.md [hunk 2]
```diff
@@ -108,10 +104,75 @@ To use tabbed application mode, developers need to opt their apps in by setting
...
+- `"absent"` to tell the browser to not show a home tab.
+- An object with two sub-properties:
+  - `"url"` with the allowed values of `"auto"` or a URL to lock the home tab to.
+  - `"icons"` with the allowed values of `"auto"` or an array of icons as in the main
+    [`"icons"` property](/add-manifest/#icons).
+
+### New tab button
+
+The new tab button, if present, should open a new tab at a URL that is within the scope of the app.
+The app may choose to set a custom URL and icon for this button. Browsers can decide how to handle
```
Comment (jpmedley) [inline src/site/content/en/blog/tabbed-application-mode/index.md:135]: >custom URL

Based on the previous statement, I'd think this goes without saying. Am I missing something?

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/tabbed-application-mode/index.md (hunk 2 above)
Comment (tomayac) [reply src/site/content/en/blog/tabbed-application-mode/index.md:135]: It's a different context, just the behavior happens to be the same.

File: src/site/content/en/blog/tabbed-application-mode/index.md (hunk 1 above)
Comment (jpmedley) [reply src/site/content/en/blog/tabbed-application-mode/index.md:114]: What is the issue here?

File: src/site/content/en/blog/tabbed-application-mode/index.md (hunk 2 above)
Comment (jpmedley) [reply src/site/content/en/blog/tabbed-application-mode/index.md:135]: If custom URL = out of scope URL, you should make the clear.

File: src/site/content/en/blog/tabbed-application-mode/index.md [hunk 3]
```diff
@@ -108,10 +104,75 @@ To use tabbed application mode, developers need to opt their apps in by setting
...
+dragging these tabs around to create new windows or combine with browser tabs.
+
+The allowed values for `"new_tab_button"` are:
+
+- `"auto"` to let the browser determine what to do.
+- `"absent"` to tell the browser to not show a new tab button.
+- An object with two sub-properties:
+  - `"url"` with the allowed values of `"auto"` or a URL to open new tabs on.
```
Comment (tomayac) [inline src/site/content/en/blog/tabbed-application-mode/index.md:143]:
```suggestion
  - `"url"` with the allowed values of `"auto"` or an in-scope URL to open new tabs on.
```

File: src/site/content/en/blog/tabbed-application-mode/index.md (hunk 2 above)
Comment (tomayac) [reply src/site/content/en/blog/tabbed-application-mode/index.md:135]: Added that the URL needs to be in-scope.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**